subprojects/packagefiles/libffmpegthumbnailer/meson.build

```diff
@@ -0,0 +1,71 @@
+project('ffmpegthumbnailer', 'cpp',
+  version : '2.2.2',
```

Comment:
Currently uses `meson_version: '>=0.47.0'`, could we add this explicitly?

Reply:
Done.